docs/resources/Channel.md

```diff
@@ -138,11 +138,17 @@ Represents a message sent in a channel within Discord.
 | Field | Type | Description |
 |-------|------|-------------|
 | title | string | title of embed |
-| type | string | type of embed |
```

Comment:
-| author | a [user](#DOCS_USER/user-object) object | the author of this message |